src/test/lambda/extensionManifest.test.ts

```diff
@@ -0,0 +1,52 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import * as packageJson from '../../../package.json'
+import * as packageNlsJson from '../../../package.nls.json'
+import * as lodash from 'lodash'
+
+describe('Test Copy Lambda Func URL in extension manifest (package.json)', async () => {
```

Comment:
I wonder if we should just test the very minimum amount of functionality here that we can, that way we aren't tying the package.json for lambdaURL directly to this test. And when one changes we have to change this and vice versa.

E.g. testing if the command title exists, it's disabled in the command palette (this could be useful for creating tests in C9, where we could open a test file and see the list of things disabled), testing that the command exists in the menu without testing the entirety of what is contained

It technically is a bit looser of a test but i'm not sure if its worth it for us to eventually maintain strict tests for everything that goes into the package.json

Reply:
> It technically is a bit looser of a test but i'm not sure if its worth it for us to eventually maintain strict tests for everything that goes into the package.json

Yeah we generally want tests to not necessarily be lenient but rather precise in what they're testing. A test for command declaration shouldn't fail if I change the command title as that's not what I'm testing.